.husky/pre-commit

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
+echo "\n============================ \n ⚙️ Running pre-commit hooks \n============================ \n"
+echo "--- Linting code and checking for errors ---\n"
+
+yarn format && yarn lint
```

Comment:
Lints and formats the whole codebase to look for any potential errors. This gets triggered on every commit. On the pre-commit lifecycle